parsers/US_NY.py

```diff
@@ -8,13 +8,21 @@
 import pandas as pd
 
 mapping = {
-           'Dual Fuel': 'unknown',
+           'Dual Fuel': 'gas', # NYISO states "Power produced by facilities capable of using natural gas and/or other fuels"
+                # <5% is oil burn from 2016 State of NYISO Market Report
            'Natural Gas': 'gas',
            'Nuclear': 'nuclear',
-           'Other Fossil Fuels': 'unknown',
-           'Other Renewables': 'unknown',
-           'Wind': 'wind',
-           'Hydro': 'hydro'
+           'Other Fossil Fuels': 'coal', # NYISO states "Power produced by either coal or oil" - assume coal, oil only power plants only as last resort
```

Comment:
Any source to document that oil is only being used as last resort?

Reply:
it is buried in the 500+ pages State of the NYISO Market - there was a chart showing oil burn went to <5% from 2015 to 2016 of all "dual fuel" capacity plants throughout the respective year.  "as a last resort" might be happening right now due to the cold snap, constrained NG pipelines, not enough NG put into storage before the winter heating season & LNG production increasing.  I will go dig it out, but this is backend information.